plugin.cmake

```diff
@@ -53,6 +53,11 @@ add_python_test(glcm
     PLUGIN HistomicsTK
 )
 
+add_python_test(segmentation_label
+    SUBMODULE TraceBoundsTest
+    PLUGIN HistomicsTK
+)
+
```

Comment:
Can you try this:

add_python_test(segmentation_label
    SUBMODULE TraceBoundsTest
    DBNAME core_segmentation_trace_bounds
    PLUGIN HistomicsTK
)

I am guessing the test name is getting too long. With `DBNAME` you can override and give it a shorter name.